throttle/tokenbucket_test.go

```diff
@@ -10,7 +10,7 @@ type TestBucket struct {
 	TokenBucket
 }
 
-func (b *TestBucket) hasExactly(tokens uint32) bool {
+func (b *TestBucket) hasExactly(tokens float64) bool {
```

Comment:
Now that tokens are being tracked as floats (which I agree was the right thing to do), exact values are less likely, especially with fractional refill rates. Where and how is hasExactly used?